ContentSearch/core/EvoContentSearch.php

```
@@ -330,26 +330,108 @@ private function likeQuery($keyword) {
+     * - フィールド重み付け（タイトル、ディスクリプション、本文冒頭、本文）
+     * - キーワード密度
+     * - 文書の長さによる正規化
+     * - 新しさ（フレッシュネス）
+     * - 短いキーワードの誤ヒット対策
+     *
+     * @param string $keyword 検索キーワード
+     * @return string SQL式
+     */
+    private function generateRelevanceScore($keyword) {
+        $escaped = db()->escape($keyword);
+        $keywordLen = mb_strlen($keyword);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$keywordLen'. (undefined)

(UnusedLocalVariable)